plf-assemblies/src/main/resources/assemblies/plf-copy-libraries-component.xml

```diff
@@ -98,7 +98,7 @@
         <!-- Avoid any direct addon inclusion on the distribution packages -->
         <exclude>org.exoplatform.addons.*:*</exclude>
       </excludes>
-      <outputFileNameMapping>${artifact.artifactId}-${artifact.baseVersion}.${artifact.extension}</outputFileNameMapping>
+      <outputFileNameMapping>${artifact.artifactId}.${artifact.extension}</outputFileNameMapping>
```

Comment:
The version is set on purpose in the jar names in order to easily identify it. Why do we need to remove them ?

Reply:
In order to detect any potential conflict between distrib and addons about third party libraries very soon in development process (when we attempt to install addon defining a JAR file that is already added in distribution, the addon installation fails).
Before, we was detecting such a problem very late in development lifecycle. In fact, it's detected when a functional bug is detected with a stack trace about JARs incompatibility. This modification will let us identify such a problem very easily and quickly.

Reply:
I thought we already decided to keep versions in jars : https://jira.exoplatform.org/browse/PLF-6294

Reply:
it's not a decision, it was a discussion that didn't finished yet, now it's up to us to decide ;-)

Reply:
so, does my arguments ans [support guys arguments](https://jira.exoplatform.org/browse/PLF-6294?focusedCommentId=496662&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-496662) convinced you ?

> set on purpose in the jar names in order to easily identify it
you mean easily identify its version ? It's already easy to do by opening the MANIFEST file from jar file by reading the file DEPENDENCIES.txt in the root folder of TOMCAT.

Reply:
> It is easier for them to find conflicts.

I think it's easier to find conflict when the version isn't added in jars in build time.
I 'm talking about automatic CI that will fail when addons will provide existing JARs. And for you instead, you are talking about manual and human readability of version from file names.

> I remember it was raised by several customers using jboss

do you have some references of that to see what's the provided arguments ?

Reply:
this is very useful for "continuous integration". As you can see in this build for example https://ci.exoplatform.org/job/FB/job/platform-private-distributions-lightweight-fb-ci/292/console

the build failed after having modified some dependencies.

>      [exec] [31m[ERROR] File jboss-logging-annotations.jar already exists.[m
>      [exec] [31m[ERROR] Installation aborted. Use --conflict=skip or --conflict=overwrite option to install it.[m

So I know now, that I should fix a dependency to not import it twice , or worse, with a different version.

Reply:
ok, so, as decided, I have reverted this change.